html/changelogs/DrInti.yml

```diff
@@ -0,0 +1,5 @@
+author: DrCelt / Intigracy
+delete-after: True
+changes: 
+- bugfix: "(DrCelt) As an effect of a bugfix, all species are now a little less weak/strong against damage types that are modified for them. Think plasmamen burn damage resistance."
```

Comment:
that's a stupid way to word that.
Fixed damage species modifiers not applying itself twice.